libs/api-mocks/msw/handlers.ts

```diff
@@ -223,6 +223,53 @@ export const handlers = makeHandlers({
 
     return 204
   },
+  floatingIpCreate({ body, query }) {
+    const project = lookup.project(query)
+    errIfExists(db.floatingIps, { name: body.name })
+
+    const newFloatingIp: Json<Api.FloatingIp> = {
+      id: uuid(),
+      project_id: project.id,
+      ip: `${[...Array(4)].map(() => Math.floor(Math.random() * 256)).join('.')}`,
```

Comment:
Nice. Because of the `join` it's already a string, so you don't need the interpolation.

```suggestion
      ip: [...Array(4)].map(() => Math.floor(Math.random() * 256)).join('.'),
```